include/deal.II/grid/manifold.h

```diff
@@ -269,6 +269,19 @@ class Manifold : public Subscriptor
                                        const Point<spacedim> &candidate) const;
 
   /**
+   * Given a point which lies close to the given manifold and to the given
+   * object, return one which lies on the Manifold.
+   *
+   * Internally this method calls the function project_to_manifold()
+   * passing the vertices of the object.
+   */
+  template<typename OBJ>
+  Point<spacedim>
+  project_to_manifold (const OBJ &obj,
```

Comment:
Can you document which kind of thing you allow as valid types for `OBJ`? I imagine these should be iterators of some kind, but it's not clear to me.